Ginger/Ginger/BusinessFlowPages_New/ActionsListViewPage.xaml.cs

```diff
@@ -168,8 +168,36 @@ private void listActions_ItemDropped(object sender, EventArgs e)
                 Act instance = null;
                 if (droppedItem is Act)
                 {
-                    Act a = droppedItem as Act;
-                    instance = (Act)a.CreateInstance(true);
+                    Act selectedAction = droppedItem as Act;
+                    if (selectedAction.AddActionWizardPage != null)
```

Comment:
we should not have duplicated code for adding Action into Actions list like we have here and in ActionsLibrarayNavPage, put the code in a method called "AddActionsHandler()" on some class, make it static, provide to it the actions to add and actions list to add to and it will do the rest.
so that way we will need to maintain only one place which will work for D&D and add from NavPage.
let me know if not clear